SimplenoteUITests/AllNotes.swift

```diff
@@ -7,24 +7,31 @@ class AllNotes {
     }
 
     class func isOpen() -> Bool {
+		//Warning for future: this works only for 'All Notes' obviously
+		//Will not work when a tag is selected.
         return app.navigationBars[UID.NavBar.allNotes].exists
     }
 
     class func open() {
         guard !isOpen() else { return }
 
-        app.navigationBars.element.buttons[UID.Button.menu].tap()
+		Sidebar.open()
```

Comment:
🤔 looks like this is something that happened earlier than this PR, too. I went back to #1171 and I can see them there, too –not sure how I missed them...

Definitely something to address in a dedicated PR, then.

Reply:
I think this is the setting you should tweak

<img width="942" alt="Screen Shot 2021-02-26 at 3 03 59 pm" src="https://user-images.githubusercontent.com/1218433/109253466-fb592f00-7843-11eb-9e0a-4bc8abdcf5d5.png">

Reply:
This was my fault: I intentionally changed spaces to tabs in the settings above some time ago, hoping it will help with trailing whitepaces. But I had to use another setting from 'Editing' tab for this. Changed. Thanks!